DEVELOPMENT.md

```diff
@@ -13,12 +13,14 @@ tox
 
 ## Building the Docs
 
-We use sphinx to build the documentation. To do this, navigate into pyQuil's top-level directory and run:
+We use sphinx to build the documentation. Before building the docs, you must have
```

Comment:
Should we add pandon to the `# docs` section of `requirements.txt`? (And then note here that they should `pip install -r requirements.txt`.)

Reply:
pandoc isnt a python package unfortunately -- needs to be installed with brew or the like

Reply:
It shows up in a pip search:
```
$ pip search pandoc
pandoc (1.0.2)                         - Pandoc Documents for Python
```

Reply:
Although maybe that's just some library interface.

Reply:
ooo interesting

Reply:
unclear if that gives you the binary though

Reply:
not looking like it does

Reply:
Bummer. Nevermind then.